src/components/select/SelectConnected.tsx

```diff
@@ -133,8 +135,14 @@ export class SelectConnected extends React.Component<ISelectProps & ISelectSpeci
             const newChildren = React.Children.map(this.props.children, (child: React.ReactElement<any>) => {
                 if (child) {
                     return React.cloneElement(child, {
-                        onKeyDown: (e: React.KeyboardEvent<HTMLElement>) => this.onKeyDown(e),
-                        onKeyUp: (e: React.KeyboardEvent<HTMLElement>) => this.onKeyUp(e),
+                        onKeyDown: (e: React.KeyboardEvent<HTMLElement>) => {

```

Comment:
I need to do this to avoid an overwrite from the parent component. I want to call the child props before the new parent keyUp function